.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'es5',
+  printWidth: 120,
+  arrowParens: 'avoid',
```

Comment:
I prefer the other style, but this probably more closely aligned with what is in the repo already